### ISSUE TITLE
Data received in foreground differs between platforms (android and ios)

### ISSUE THREAD

**Issue by mreis1** [fechanique/cordova-plugin-fcm#289]
If you notice android and ios foreground examples, you will see that only iOS provides full access to the original push notification content. 

**Why do I care about the content while in foreground?**
By having access to the original notification content I internally show a page with unseen pushed notifications. 

**My workaround** 
Send a duplicate of title and body on our push "data" object.

iOS version: 8.x
Android Vesion: 7.x

![fcmplugin](https://cloud.githubusercontent.com/assets/1719667/25187086/559a46f8-2522-11e7-9479-5fb48d856093.png)

**Comment by arindamnayak** [comment]
I think this is mentioned in a pull request #284. Once this is merged, it will be solved.
